docs/apps/sole-factory.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 1
+title: Macrometa Sole Factory
+---
+
+## Macrometa GDN
+
+| **Platform**                       | **Tenat**                      | **Geo Fabric** |
+| ---------------------------------- | ------------------------------ | -------------- |
+| [Play](https://play.macrometa.io/) | `demo@macrometa.com` | `e-commerce-search-flow` |
+
+## Deployment
+
+| **Source Code**                       | **Deployment**                      | 
+| ---------------------------------- | ------------------------------ | 
+| [Github](https://github.com/Macrometacorp/e-commerce) | [Deployment](https://macrometacorp.github.io/e-commerce/) | 
+
+## Solution
+
+Experience the full potential of Macrometa's advanced full-stack capabilities with our innovative e-commerce demo. Discover a fully functional e-commerce website that offers personalized experiences to every visitor. Our cutting-edge technology tracks and analyzes visitor actions, enabling us to provide tailored recommendations based on their preferences. Furthermore, our e-commerce store boasts real-time stock updates and features a sophisticated search engine, ensuring a seamless and efficient shopping experience.
+
+## Components
+
+This demo includes the following components:
+
+- Personalized recommendations - Macrometa Stream Workers read visitor actions from a Collection Stream, creating a list of recommended products for each visitor.
```

Comment:
Nothing in this list should be capitalized except Macrometa and the first word in the sentence or definition term. Remove GDN.